mriqc/workflows/anatomical.py

```diff
@@ -845,6 +834,102 @@ def airmsk_wf(name="AirMaskWorkflow"):
     return workflow
 
 
+def synthstrip_wf(name="synthstrip_wf", omp_nthreads=None):
+    """Create a brain-extraction workflow using SynthStrip."""
+    from nipype.interfaces.ants import N4BiasFieldCorrection
+    from niworkflows.interfaces.nibabel import IntensityClip, ApplyMask
+    from mriqc.interfaces.synthstrip import SynthStrip
+
+    inputnode = pe.Node(niu.IdentityInterface(fields=["in_files"]), name="inputnode")
+    outputnode = pe.Node(
+        niu.IdentityInterface(
+            fields=["out_corrected", "out_brain", "bias_image", "out_mask"]
+        ),
+        name="outputnode",
+    )
+
+    # truncate target intensity for N4 correction
+    pre_clip = pe.Node(IntensityClip(p_min=10, p_max=99.9), name="pre_clip")
+
+    pre_n4 = pe.Node(
+        N4BiasFieldCorrection(
+            dimension=3,
+            num_threads=omp_nthreads,
+            rescale_intensities=True,
+            copy_header=True,
+        ),
+        name="pre_n4",
+    )
```

Comment:
I know we are only certain this will work with humans, but thinking forward, would it be possible to make the n4 more configurable by adding function parameters? i.e. maybe setting bspline grids?

Reply:
Sure, but first we will need to make sure SynthStrip works on rodents...

Reply:
Right, not sure how that would generalize. Might do OK, but not perfect. A multi-species training set would be a good next step though